app/controllers/users/phones_controller.rb

```diff
@@ -12,9 +12,13 @@ class PhonesController < ApplicationController
     before_action :confirm_recently_authenticated_2fa
 
     def add
-      user_session[:phone_id] = nil
-      @new_phone_form = NewPhoneForm.new(user: current_user, analytics: analytics)
-      analytics.add_phone_setup_visit
+      if IdentityConfig.store.consolidate_phone_controllers
```

Comment:
I'm not entirely clear why we have the feature flag, if we're planning to update all the links to route to `PhoneSetupController`. Should we just leave this controller as-is, remove the feature flag, and update all the existing routes to point to `phone_setup_path`, then the follow-up pull request can remove this controller after the next deploy?

Reply:
I think I had this to avoid the 50/50 state but I think ur right we might not need this if users are being redirected to existing PRs.